<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>calc2.ico</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="MemSub">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #f4d5a8; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="MemRecall">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #f4d5a8; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>MR</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Multiply">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #ebb1b1; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="Add">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #ebb1b1; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #f4d5a8; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Divide">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #ebb1b1; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="AllClear">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="Subtract">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #ebb1b1; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="MemAdd">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #f4d5a8; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: #C0C0C0; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}
QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="memlabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
